### PR TITLE
adding my bios

### PR DESCRIPTION
**Pull request by sktay**
now it worked @halfaro1

### CODE REVIEW

**Comment by lucy-sheridan** [comment]
hello world

**Comment by githubteacher** [comment]
@sktay great!

File: sktay-bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### Hometown
+- San Jose
+
+### What else?
```
Comment (githubteacher) [inline sktay-bio.md]: Anything you like, @sktay!

**Comment by sclouallstate** [comment]
Nice job @sktay !

File: sktay-bio.md (hunk 1 above)
Comment (jfreise) [inline sktay-bio.md]: @sktay Adding a line comment!

**Pull request merged**